modules/ROOT/pages/_partials/data-sync/ref_terminology.adoc

```diff
@@ -1,14 +1,14 @@
 //':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
-[id='ref_terminology_{context}']
+[id='ref-data-sync-terminology-{context}']
+= Data Sync terminology
 
-= {sync-service} Terminology
+This section describes terminology that is associated with Data Sync.
 
-This section describes terminology that is associated with {sync-service}.
+.Data Sync terminology
```

Comment:
duplicate of heading

Reply:
Changing this to "Data Sync terms" in the incoming PR - feel free to enhance :+1: